src/_tutorials/server/cmdline.md

```diff
@@ -138,7 +144,7 @@ But for this first app, let's do the minimum necessary to get the code to run:
 Once you have your app's dependencies,
 you can run the app from the command line over any text file,
 like `pubspec.yaml` or `quote.txt`
-([downloadable file]({{site.repo.this}}/blob/master/examples/misc/test_data/quote.txt?raw=1)):
+([downloadable file]({{site.repo.this}}/blob/master/null_safety_examples/misc/test_data/quote.txt?raw=1)):
 
 ```terminal
 $ dart run dcat.dart -n quote.txt
```

Comment:
This command didn't work for me because I was in the top-level directory. We could either (1) tell them to `cd bin` or (2) use one of the following commands (updating the no-arg example to match), and update the command-line arg figure below (maybe as an SVG, so it'll be clearer and easier to update in the future). (1) is much easier. 🤷‍♀️ 

How I did this from the top directory:

- `dart run dcat -n quote.txt`
- `dart run bin/dcat.dart -n quote.txt`

Reply:
Ah sorry, I unconsciously added the bin/ when testing. I think adding the `/bin` may be the most appropriate for this case, but let me know if you think just the name is better.

As for updating the image, I can also create an SVG. Any particular tool you tend to use?

Reply:
I'm good with adding the `bin/`. It makes things nice and clear, even if it's not the shortest possible way.

As for the image, the current one's in Omnigraffle, but I've been creating new images using diagrams.net (aka draw.io).

Reply:
Never used either before, so I gave diagrams.net a shot, let me know what you think.